### ISSUE TITLE
error: internal compiler error: prepared statement had 2 statements, expected 1

### ISSUE THREAD

**Issue by simplenotezy** [JasonShin/sqlx-ts#79]
Trying to set it up for a Nest.js project, it fails with the following error:

```
❯ npx sqlx-ts src/ --db-host=127.0.0.1 --db-port=26257 --db-type=postgres --db-user=root
[INFO] Scanning "src/" for SQLs with extension Ts
error: internal compiler error: prepared statement had 2 statements, expected 1
    --> <src/database/cli/seed.command.ts>:1728:3
     |
1728 | /   return dataSource.query(`
1729 | |   insert into public.interest_category (name)
1730 | |   values  ('Creativity'),
1731 | |           ('Food & Drinks'),
...    |
1813 | |         ('Yoga', 'Sports');
1814 | |   `);
     | |_____^

**Comment by JasonShin** [comment]
that's annoying, I will simply find a demo nest.js project to test against. By any chance, is the code that you are testing against an open-source project? (if so I can test against that)

**Comment by JasonShin** [comment]
Hello, could you revise the SQL that you have in this example?

I see that this is an INSERT query against `interest_category` table with target column name `name`, however, the values that you are trying to insert does not match
`('Yoga', 'Sports')` are two values but the SQL is looking for just a single value.

Also, could you provide an example of this code (if possible?) because sqlx-ts wouldn't be able to pick up code that is not contained inside `sql` tagged template string (for example `sql\`SELECT id FROM items\``)

As an example of working insert statement, here is a screenshot:
<img width="695" alt="Screen Shot 2023-10-22 at 11 28 42 am" src="https://github.com/JasonShin/sqlx-ts/assets/2525002/79f9ca95-c812-4100-9a3c-0a1402492382">

**Comment by JasonShin** [comment]
Also, I will add a testing against nest.js source code in https://github.com/JasonShin/sqlx-ts-demo

**Comment by simplenotezy** [comment]
Hey @JasonShin sure thing! This is the code that it picked up:

Line 1727:
```ts
const createInterests = (dataSource: DataSource) => {
  return dataSource.query(`
  insert into public.interest_category (name)
  values  ('Creativity'),
          ('Food & Drinks'),
          ('Going out'),
          ('Movies & TV'),
          ('Music'),
          ('Pets'),
          ('Sports'),
          ('Trips');

  insert into public.interest (name, category)
values  ('A sweet tooth', 'Food & Drinks'),
        ('Acting', 'Creativity'),
        ('Action', 'Movies & TV'),
        ('Animated', 'Movies & TV'),
        ('Anime', 'Movies & TV'),
        ('Art', 'Creativity'),
        ('Athletics', 'Sports'),
        ('Backpacking', 'Trips'),
        ('Badminton', 'Sports'),
        ('Bar', 'Going out'),
        ('Beach holiday', 'Trips'),
        ('Beer', 'Food & Drinks'),
        ('Big city holiday', 'Trips'),
        ('Birds', 'Pets'),
        ('Cats', 'Pets'),
        ('Caviar', 'Food & Drinks'),
        ('Champagne', 'Food & Drinks'),
        ('Classic', 'Music'),
        ('Coffee', 'Food & Drinks'),
        ('Comedy', 'Movies & TV'),
        ('Cycling', 'Sports'),
        ('Design', 'Creativity'),
        ('Dogs', 'Pets'),
        ('Doubbledate', 'Going out'),
        ('Drama', 'Movies & TV'),
        ('Drinks', 'Going out'),
        ('Fest', 'Going out'),
        ('Festival', 'Going out'),
        ('Fish', 'Pets'),
        ('Fitness', 'Sports'),
        ('Foodie', 'Food & Drinks'),
        ('Football', 'Sports'),
        ('Golf', 'Sports'),
        ('Gymnastics', 'Sports'),
        ('Handball', 'Sports'),
        ('Handiwork', 'Creativity'),
        ('Hiking', 'Trips'),
        ('Hip hop', 'Music'),
        ('Horror', 'Movies & TV'),
        ('House', 'Music'),
        ('In', 'Creativity'),
        ('Jazz', 'Music'),
        ('Meditation', 'Sports'),
        ('Netflix', 'Movies & TV'),
        ('Nightclub', 'Going out'),
        ('Photography', 'Creativity'),
        ('Pop', 'Music'),
        ('R&B', 'Music'),
        ('Rabbit', 'Pets'),
        ('Rap', 'Music'),
        ('Road trip', 'Trips'),
        ('Rock', 'Music'),
        ('Punk', 'Music'),
        ('Punk rock', 'Music'),
        ('Hardcore punk', 'Music'),
        ('Skate punk', 'Music'),
        ('Romantic', 'Movies & TV'),
        ('Rowing', 'Sports'),
        ('Run', 'Sports'),
        ('Sci-fi', 'Movies & TV'),
        ('Singing', 'Creativity'),
        ('Skateboard', 'Sports'),
        ('Ski', 'Sports'),
        ('Ski holiday', 'Trips'),
        ('Snowboard', 'Sports'),
        ('Swimming', 'Sports'),
        ('Table tennis', 'Sports'),
        ('Tennis', 'Sports'),
        ('Theater', 'Creativity'),
        ('Vegan', 'Food & Drinks'),
        ('Vin', 'Food & Drinks'),
        ('Volleyball', 'Sports'),
        ('Write', 'Creativity'),
        ('Yoga', 'Sports');
  `);
};
```

**Comment by JasonShin** [comment]
Hello @simplenotezy, I've released more fix to enhance support for nest.js project source code and it was released in 0.7.0. I figured out some TS syntax wasn't picked up via sqlx-ts in nest.js codebase

**Comment by simplenotezy** [comment]
> Hello @simplenotezy, I've released more fix to enhance support for nest.js project source code and it was released in 0.7.0. I figured out some TS syntax wasn't picked up via sqlx-ts in nest.js codebase https://github.com/JasonShin/sqlx-ts/pull/80.
> 
> 
> 
> In regards to the query above in your last example, I will try to reproduce it on my end. However, that query is not even contained inside `sql` tag, so I highly doubt that's the problematic query 🤔 

Amazing!

And regarding the last part; sure it's strange, but I just copy pasted the error output, and that's indeed the line 🤔

**Comment by JasonShin** [comment]
I will close this for now hoping that it is fixed with the latest updates
